### ISSUE TITLE
Enemy has same color as Player in MiniMap

### ISSUE THREAD

**Issue by MooDySmiles** [RafaelBarbosatec/bonfire#413]
When setting `playerColor` in the `miniMap` widget, enemies are colored the same way due to `componentsRenderDefault` logic at row `178`

```dart
} else if (component is Enemy) {
  component.renderCollision(
    canvas,
    widget.playerColor ?? Colors.red.withOpacity(0.5),
  );
}
```

It should be `widget.enemyColor` and not `widget.playerColor`.

**Comment by RafaelBarbosatec** [comment]
Fixed in version `2.12.6`

**Comment by MooDySmiles** [comment]
Thanks now works as expected!
